### DISCUSSION TITLE
Pn geometric properties

### DISCUSSION THREAD

**Discussion by kizzle4shaw** [PMEAL/OpenPNM#2745] [Q&A] [answered]
I have extracted a network from Massagis jaune carbonate using the snow2 algorithm. I Was trying to display the networks. geometrical properties such are the pore radius and throat distribution, but I'm getting something unexpected. Could someone please help me understand the problem here.
![IMG_8497](https://user-images.githubusercontent.com/32852010/234219620-64670677-47d5-432c-b779-0ae814554acc.jpg)

**Comment by jgostick** [comment, marked as answer]
Probably you bins are too wide and are overlapping?

**Reply by kizzle4shaw** [reply]
Yes, thank you Professor.
